### PR TITLE
Milestone - Project Completed

### PR DESCRIPTION
**Pull request by NicolasAndrehh**
# Milestone - Project completed
This pull request includes changes that meet the following requirements:

### Project Requirements
- Create a new Rails app called 'hello-rails-react'.
   - Use the command rails new -j webpack my_new_rails_app as described in the previous lesson's tutorial.
- Verify that the jsbundling-rails gem is added to your Gemfile.
- Initialize your project with Git.
- Install needed node dependencies as described in the previous lesson's tutorial.
- Create a babel.config.js in the root folder of your project as described in the previous lesson's tutorial.
- Update webpack.config.js to include the babel loader as described in the previous lesson's tutorial.
- Make sure that your project has a Postgres database set up.
- Create a table for storing your messages and create 5 different greetings.
- Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and Rails route).
- Create a static view that will be the root of your app.
- Create the App component with react-router and render it in your static view.
- Create the Greeting component that will display a greeting. Set it up as a route in your App component.
- Create a store, an action, and a reducer that will connect to your API endpoint to get the random greeting.
- Display the random greeting in your Greeting component.

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>RailsReactWebpackIntro</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
```
Comment (brytebee) [inline app/views/layouts/application.html.erb:10]: image

I keep getting the error attached above, kindly fix it.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @NicolasAndrehh 👋,

##

### Highlights💡
- Great project structure and setup.✔️
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (NicolasAndrehh) [reply app/views/layouts/application.html.erb:10]: Hey @brytebee, I just cloned my project and I figured out that you're right, I'm getting the same error but I solve it just running the following commands:
```shh 
rails assets:precompile
```
And this one for populate the database with the greetings:
```shh 
rails db:migrate
```
Could you try it, please?

Comment (Whoistolu) [reply app/views/layouts/application.html.erb:10]: - [ ] Well resolved.
Screenshot 2023-07-11 at 18 08 50

**Review by Whoistolu — APPROVED** [review]
### Hi @NicolasAndrehh 

Great job implementing the fixes recommended by the previous reviewer 👍🏽 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**